firebase-database-behavior.html

```diff
@@ -30,11 +30,11 @@
 
         /**
          * Path to a Firebase root or endpoint. N.B. `path` is case sensitive.
+         * @type {string|null}
          */
         path: {
           type: String,
-          observer: '__pathChanged',
-          value: null
+          observer: '__pathChanged'
```

Comment:
Why was the default value removed? This is a breaking change, yet it should fit in the same spec. Also will `{String}` work since it's nullable unlike `{string}`